todo/src/scripts/storageController.js

```diff
@@ -0,0 +1,88 @@
+class StorageController {
+    constructor(name) {
+        this.init();
+        this.data;
+    }
+
+    init() {
+        if (!localStorage.getItem('toDo')){
+            localStorage.setItem('toDo', JSON.stringify(
+                {
+                    'currentTasks': [],
+                    'completeTasks': [],
+                }
+            ));
+        }
+        this.data = JSON.parse(localStorage.getItem('toDo'));
+        this.removeEmptyItems();
+    }
+
+    removeEmptyItems() {
+        this.data['currentTasks'] = this.data['currentTasks'].filter((item) => {return item != null});
+        this.data['completeTasks'] = this.data['completeTasks'].filter((item) => {return item != null});    
+    }
+
+    updateStorage() {
+        localStorage.setItem('toDo', JSON.stringify(this.data));
+        this.data = JSON.parse(localStorage.getItem('toDo'));
+    }
+    
+    getItemsList() {
+        return this.data;
+    }
+
+    getCurentItemsList() {
+        return this.data['currentTasks'];
+    }
+
+    getCompleteItemsList() {
+        return this.data['completeTasks'];
+    }
+
+    addNewItem(title, task, priority, date) {
+        this.data.currentTasks.push(
+            {
+                'title': title,
+                'task': task,
+                /* 'date': date, */
+                'priority': priority,
+                'date': date
+            }
```

Comment:
here you should create item with Task constructor if you write code in OOP style

Reply:
it was my mistake to write in OOP style.